packages/components/src/internal/components/base/Grid.tsx

```diff
@@ -322,6 +322,7 @@ export interface GridProps {
     loadingText?: React.ReactNode;
     messages?: List<Map<string, string>>;
     responsive?: boolean;
+    className?: string;
```

Comment:
I see that the use of `<Grid>` in `DomainPropertiesGrid` doesn't use this new property. Was it needed for some other usage?

Reply:
Ah, yes. This was before the theme/ style fix was made. Prop removed